test/unit/callbacks_and_delays_test.lua

```diff
@@ -84,3 +84,44 @@ function g.test_delays_and_scan_callbacks(cg)
 
     task:kill()
 end
+
+function g.test_error_callback(cg)
+    local space = cg.space
+    local total = 10
+    for i = 1, total do
+        space:insert({i, tostring(i)})
+    end
```

Comment:
it is actually a test setup and can be moved to `g.before_test()`.

Reply:
moved to `before_each` because `test_delays_and_scan_callbacks` have the same prepare stage